open-budget-il/main.py

```diff
@@ -677,22 +680,23 @@ def get_query(self):
         index = search.Index(name="OpenBudget")
         results = []
         try:
-            searchQueryStringList = [queryString]
```

Comment:
Since we don't specify specific filters to specific types (at least with the current implementation), why not make the query like this:
מדינה OR year=2014
?
It will give the same results as the two above queries (but with a much simpler query).